test/e2e/testing-helpers.js

```diff
@@ -0,0 +1,47 @@
+import { ClientFunction } from 'testcafe'
+
+export async function assertTable (t, table, expectedData) {
+    const columns = ['name', 'power']
+
+    await t.expect(table.find('td').count, expectedData.length * columns.length)
+
+    let currentTdText = null
+    let expectedText = null
+
+    for (let i = 0; i < expectedData.length; i++) {
+        for (let j = 0; j < columns.length; j++) {
+            currentTdText = await table.find('tbody').find('tr').nth(i).find('td').nth(j).textContent
+            expectedText = expectedData[i][columns[j]]
+
+            await t.expect(currentTdText).contains(expectedText)
+        }
+    }
+}
+
+export async function assertPoligonPoints (t, countPoint) {
+    const checkPolygonPoints = ClientFunction(() => {
+        /* eslint-disable no-undef */
+        var points = stats.map(function (stat, i) {
+            var point = valueToPoint(stat.value, i, countPoint)
+            return point.x + ',' + point.y
+        }).join(' ')

```

Comment:
@miherlosev has applied eslint rules from vuejs repo in the last commit. It seems they differ from ours.